packages/react/src/runtimes/remote-thread-list/adapter/cloud.tsx

```
@@ -0,0 +1,99 @@
+    generateTitle: async (threadId, messages) => {
+      return cloud.runs.stream({
+        thread_id: threadId,
+        assistant_id: "system/thread_title",
+        messages: toCoreMessages(messages),
+      });
+    },
+
+    unstable_Provider: useCallback<FC<PropsWithChildren>>(({ children }) => {
+      const history = useAssistantCloudThreadHistoryAdapter({
+        get current() {
+          return adapterRef.current.cloud!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)